### DISCUSSION TITLE
Bifrost for maya_usd-0.25?

### DISCUSSION THREAD

**Discussion by sanders3d** [Autodesk/maya-usd#3450] [General] [answered]
As the topic says, we just installed Bifrost but it seems only to be compatible with 0.23? Is there any trick in getting 0.25 to work?

regards
stefan

**Comment by sepux** [comment, marked as answer]
Bifrost 2.8 supports 0.25 

Also
https://github.com/Autodesk/bifrost-usd/releases/tag/v1.2.1
